go.mod

```diff
@@ -8,11 +8,11 @@ require (
 	github.com/google/go-cmp v0.5.6 // indirect
 	github.com/google/go-github/v29 v29.0.3 // indirect
 	github.com/google/go-github/v32 v32.1.0
```

Comment:
i think we need to clean these older version sometime.

Reply:
29 is used indirectly somewhere as a sub-dep. 32 is used by scorecards. I'll make an issue there.